browsers/firefox_android.json

```diff
@@ -54,237 +54,237 @@
         "31": {
           "release_date": "2014-07-22",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/31",
+          "release_notes": "https://www.mozilla.org/firefox/android/31.0beta/releasenotes/",
           "status": "retired"
         },
         "32": {
           "release_date": "2014-09-02",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/32",
+          "release_notes": "https://www.mozilla.org/firefox/android/32.0beta/releasenotes/",
           "status": "retired"
         },
         "33": {
           "release_date": "2014-10-14",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/33",
+          "release_notes": "https://www.mozilla.org/firefox/android/33.0beta/releasenotes/",
           "status": "retired"
         },
         "34": {
           "release_date": "2014-12-01",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/34",
+          "release_notes": "https://www.mozilla.org/firefox/android/34.0beta/releasenotes/",
           "status": "retired"
         },
         "35": {
           "release_date": "2015-01-13",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/35",
+          "release_notes": "https://www.mozilla.org/firefox/android/35.0beta/releasenotes/",
           "status": "retired"
         },
         "36": {
           "release_date": "2015-02-27",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/36",
+          "release_notes": "https://www.mozilla.org/firefox/android/36.0beta/releasenotes/",
           "status": "retired"
         },
         "37": {
           "release_date": "2015-03-31",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/37",
+          "release_notes": "https://www.mozilla.org/firefox/android/37.0beta/releasenotes/",
           "status": "retired"
         },
         "38": {
           "release_date": "2015-05-12",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/38",
+          "release_notes": "https://www.mozilla.org/firefox/android/38.0beta/releasenotes/",
           "status": "retired"
         },
         "39": {
           "release_date": "2015-07-02",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/39",
+          "release_notes": "https://www.mozilla.org/firefox/android/39.0beta/releasenotes/",
           "status": "retired"
         },
         "40": {
           "release_date": "2015-08-11",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/40",
+          "release_notes": "https://www.mozilla.org/firefox/android/40.0beta/releasenotes/",
           "status": "retired"
         },
         "41": {
           "release_date": "2015-09-22",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/41",
+          "release_notes": "https://www.mozilla.org/firefox/android/41.0beta/releasenotes/",
```

Comment:
This shoudn’t point to the beta release notes, but to the [normal release notes](https://www.mozilla.org/firefox/android/41.0/releasenotes/).